packages/core/src/components/Tipseen/Tipseen.tsx

```diff
@@ -71,7 +72,7 @@ export const TipseenContext = React.createContext<TipseenColor>(TipseenColor.PRI
 
 const Tipseen: VibeComponent<TipseenProps> & {
   closeButtonThemes?: typeof TipseenCloseButtonTheme;
-  positions?: typeof DialogPosition;
+  positions?: TooltipPositionsType;
```

Comment:
same as before, you removed the static prop from here
so either get it back there, or remove line 75
```
export default withStaticProps(Tipseen, {
  closeButtonThemes: TipseenCloseButtonTheme,
  positions: DialogPosition,
  animationTypes: AnimationType,
  justifyTypes: JustifyType,
  hideShowTriggers: HideShowEvent,
  colors: TipseenColor
});
```